src/v2/Apps/Auction2/Components/Form/OnSubmitValidationError.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react"
+import { useEffect } from "react"
+import { useFormContext } from "v2/Apps/Auction2/Hooks/useFormContext"
+
+/*
+  NOTE: Feb 2022: Not sure what this does or why
+
+  Old Comment:
+  This component exists only to capture formik's renderProps and track form
+  submission events. It essentially says:
+  - IF the form has been submitted at least once
+  - AND the form is not submitting at this moment
+  - AND the form is invalid
+  - AND (in useEffect dependencies array) the form submitting state just
+    changed (because it is false, it must have been true above)
+  - THEN run the callback prop.
+  Background:
+    https://github.com/jaredpalmer/formik/issues/1484#issuecomment-490558973
+ */
+
```

Comment:
Yeah, I find this odd — _why_ would you ever want to call `onSubmit` if there's a validation error.

Reply:
I'm tempted to just delete this because i honestly don't understand why it was added initially and we don't see this kind of thing elsewhere. I guess i will and we can work from there.